aries-site/src/layouts/content/Example/DoDontContainer.js

```diff
@@ -1,15 +1,17 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
-import { Box, Paragraph, Text } from 'grommet';
+import { Box, Paragraph, Text, ResponsiveContext } from 'grommet';
 import { Container } from '.';
 
 export const DoDontContainer = ({
   bestPractice: bestPracticeProp,
+  height: heightProp,
```

Comment:
Small comment -- do we have to rename it to `heightProp`, or can we just leave as `height` and pass it through as is?

Reply:
done